packages/@storefront/core/test/unit/core/url-beautifier/details/parser.ts

```diff
@@ -97,6 +97,23 @@ suite('DetailsUrlParser', ({ expect }) => {
     expect(parsed.variants).to.eql(expectedDetail.variants);
   });
 
+  // tslint:disable-next-line max-line-length
+  it('should extract query string parameters of a URL specifying "area" and "collection", and add them to the return object', () => {
+    const url = 'dress/293014?collection=joey&area=carlo';
+    const expectedDetail = {
+      variants: [],
+      collection: 'joey',
```

Comment:
I think the use of your names for the area and collection values might be a little *too* fun. Consider using values which approximate real area and collection names.

Reply:
lol will change.